src/kolibri2zim/scraper.py

```diff
@@ -91,6 +91,17 @@ def read_from_zip(ark, member):
     return ark.open(member).read()
 
 
+def wrap_failure_details(func):
+    def wrapper(self, item):
+        try:
+            node_id, kind = item
+            return func(self, item)
+        except Exception as ex:
+            raise RuntimeError(f"Failed to process {kind} node {node_id}") from ex
```

Comment:
```suggestion
            raise RuntimeError(f"Failed to process {kind} node {node_id}: {exc}") from exc
```

Wouldn't that be more useful? I think the `from exc` brings the stacktrace only